### PR TITLE
Add redirect for Guild admin page

### PR DESCRIPTION
**Pull request by LikkanChung**
Add redirect for committee admin page on Guild website. Makes it easier to navigate to if loggin in using mobile

### CODE REVIEW

**Comment by JTWWilson** [comment]
Sounds like you need a bookmark on your browser, not a redirect from our website. I think this is entirely the wrong use case

**Comment by LikkanChung** [comment]
> Sounds like you need a bookmark on your browser, not a redirect from our website. I think this is entirely the wrong use case

@JTWWilson Doesn't really solve the issue - this makes it easier for everyone in the future.

File: _data/redirects.yaml [hunk 1]
```diff
@@ -45,6 +45,8 @@
 # uni
 - source: /guild
   destination: https://www.guildofstudents.com/studentgroups/societies/css/
+- source: /guild/admin
```
Comment (AlexJBanks) [inline _data/redirects.yaml:48]: my concern was that you weren't going to do this (i.e. hide it in a subdirectory) but you have. So I'm satisfied

Comment (JTWWilson) [reply _data/redirects.yaml:48]: It’s hidden behind a layer of authentication so it doesn’t really matter anyway

Comment (AlexJBanks) [reply _data/redirects.yaml:48]: it would be clunky to not put it one layer behind

**Comment by AlexJBanks** [comment]
> 
> 
> > Sounds like you need a bookmark on your browser, not a redirect from our website. I think this is entirely the wrong use case
> 
> @JTWWilson Doesn't really solve the issue - this makes it easier for everyone in the future.

I'm torn as I agree with both these sentiments. It definetly is an issue that each user could sort out themselves, but we need to access the guild admin website often enough (commonly on phones) it makes sense to give a solution. That would either be a redirect via the website or a link in committee discord. I'd argue this is a lot more sensible and useable.

**Comment by Danyc0** [comment]
> > > Sounds like you need a bookmark on your browser, not a redirect from our website. I think this is entirely the wrong use case
> > 
> > 
> > @JTWWilson Doesn't really solve the issue - this makes it easier for everyone in the future.
> 
> I'm torn as I agree with both these sentiments. It definitely is an issue that each user could sort out themselves, but we need to access the guild admin website often enough (commonly on phones) it makes sense to give a solution. That would either be a redirect via the website or a link in committee discord. I'd argue this is a lot more sensible and useable.

A link in \#links sounds like a much nicer solution to this that I for some reason hadn't thought of

**Pull request closed without merging**